### DISCUSSION TITLE
Consider adding immich to list of alternatives as it's a very solid open-source clone of Google Photos

### DISCUSSION THREAD

**Discussion by bgriffen** [TheLastGimbus/GooglePhotosTakeoutHelper#221] [General]
The developers have made some amazing progress these last few months and is at a point where I'd happily introduce it to others as a genuine clone of Google Photos. [Immich here](https://github.com/immich-app/immich). I have no affiliation, just a happy user, running it on Unraid + nginx to serve publicly (username password) to my phone on the go. Has iOS support, AI, great search etc. the works.

**Comment by TheLastGimbus** [comment]
hi! forgot to write back, but i checked it out and it looks awesome!! totally adding it
